### PR TITLE
WFLY-12136 Fix consistency of HttpSessionActivationListener event timing for distributed session managers

### PR DESCRIPTION
**Pull request by pferraro**
https://issues.jboss.org/browse/WFLY-12136

### CODE REVIEW

File: clustering/web/infinispan/src/main/java/org/wildfly/clustering/web/infinispan/session/InfinispanSessionManager.java [hunk 1]
```diff
@@ -208,33 +149,28 @@ public String createIdentifier() {
     public Session<L> findSession(String id) {
         Map.Entry<MV, AV> value = this.factory.findValue(id);
         if (value == null) {
-            InfinispanWebLogger.ROOT_LOGGER.tracef("Session %s not found", id);
+            InfinispanWebLogger.ROOT_LOGGER.infof("Session %s not found", id);
```
Comment (bstansberry) [inline clustering/web/infinispan/src/main/java/org/wildfly/clustering/web/infinispan/session/InfinispanSessionManager.java]: This doesn't look like INFO.

File: clustering/web/infinispan/src/main/java/org/wildfly/clustering/web/infinispan/session/InfinispanSessionManager.java [hunk 2]
```diff
@@ -208,33 +149,28 @@ public String createIdentifier() {
...
             return null;
         }
         ImmutableSession session = this.factory.createImmutableSession(id, value);
         if (session.getMetaData().isExpired()) {
-            InfinispanWebLogger.ROOT_LOGGER.tracef("Session %s was found, but has expired", id);
+            InfinispanWebLogger.ROOT_LOGGER.infof("Session %s was found, but has expired", id);
```
Comment (bstansberry) [inline clustering/web/infinispan/src/main/java/org/wildfly/clustering/web/infinispan/session/InfinispanSessionManager.java]: This doesn't look like INFO.

**Review by bstansberry — CHANGES_REQUESTED** [review]
Another huge refactor. :( Seems ok though, logging issues aside.

**Comment by pferraro** [comment]
Fixed inadvertent logging level changes (residual debugging cruft).  Rebased against master.

**Review by bstansberry — APPROVED** [review]

**Pull request merged**